src/book/03-guides/02-tools/03-webots.mdx

```diff
@@ -363,3 +363,45 @@ From the root of the NUWebots repository run
 ```
 
 Where `<name of controller>` is the name you want to use for the new controller. A folder and files will be created with this name, so they must be valid file names (don't use spaces in your controller name).
+
+## Recording a Game
+
+NUbots runs full Webots games to benchmark the current performance of the code. The following steps describe how to create a video of a full game.
+
+1. Ensure you have followed the previous steps [Install Webots](#install-webots) and [Running the Official RoboCup Simulation in Webots](#running-the-official-robocup-simulation-in-webots).
+
+2. Set up the game configuration.
+
+   1. Open the [game.json file](https://github.com/NUbots/NUWebots/blob/main/webots/projects/samples/contests/robocup/controllers/referee/game.json).
+   2. Change the `"blue":` `"id"` to `12`.
+   3. Change the `"blue":` `config:` field to `"team_1.json"`.
+
+   At the moment, there is an issue due to the fact that both teams have the same team ID since they are both NUbots teams. A penalty to one robot will also penalise the corresponding robot on the other team.
+
+3. Open a terminal and run the following to open Webots with the Robocup world and the GameController:
+
+   ```bash
+   $WEBOTS_HOME/webots $WEBOTS_HOME/projects/samples/contests/robocup/worlds/robocup.wbt
```

Comment:
Do we need the normal environment variables defined in line, like we usually have when running webots?

Reply:
Which environment variables? I always just run `./webots` and it all works fine, unless I'm adding in the game controller filter.

Reply:
Looking at the webots guide now, I was running on outdated info. I was thinking of the variables we add to the bashrc and the UDP_FILTER - I always passed those to the webots call directly. All good